src/utilities/hw 2/getAlbumsSortedByTitle/getAlbumsSortedByTitle.test.js

```diff
@@ -0,0 +1,44 @@
+import { getAlbumsSortedByTitle } from './getAlbumsSortedByTitle';
+import { getAlbums } from './getAlbums';
+
+jest.mock('./getAlbums', () => ({
+  __esModule: true,
+  getAlbums: jest.fn(),
+}));
+
+describe('The getAlbumsSortedByTitle function', () => {
+  describe('basic behaviour', () => {
+    it('should call the getAlbums function', async () => {
+      getAlbums.mockResolvedValue([]);
+      await getAlbumsSortedByTitle();
+      expect(getAlbums).toHaveBeenCalled();
+    });
+  });
+
+  describe('when albums array is correctly fetched in getAlbums', () => {
+    const sortedArray = [
+      { id: 2, title: 'A Album' },
+      { id: 1, title: 'B Album' },
+      { id: 3, title: 'C Album' },
+    ];
+
+    it('should return properly sorted array', async () => {
+      getAlbums.mockResolvedValue([
```

Comment:
you should use the `sortedArray` variable here.
```suggestion
      getAlbums.mockResolvedValue(sortedArray);
```

Also you can think of a better name for this variable. You should always indicate what is inside this array.

Reply:
done